app/controllers/admin/tenants_controller.rb

```diff
@@ -1,5 +1,58 @@
 class Admin::TenantsController < Admin::BaseController
   def index
-    @tenants = Tenant.select(:organization, :location, :slug).all
+    if !Tenant.unapproved.empty?
+      @unapproved = Tenant.unapproved.select(:organization,
+                                             :location,
+                                             :slug,
+                                             :id).all.order(:id)
+    else
+      @unapproved = nil
```

Comment:
I have it set so the view knows when to display nil. This can probably be done better.